mesh/mesh.go

```diff
@@ -551,61 +559,32 @@ func (msh *Mesh) AddBallot(
 	ctx context.Context,
 	ballot *types.Ballot,
 ) (*wire.MalfeasanceProof, error) {
+	var (
+		initiallyMalfeasant bool
+		start               time.Time
+	)
+	initiallyMalfeasant = false
 	malicious := msh.atxsdata.IsMalicious(ballot.SmesherID)
 	if malicious {
 		ballot.SetMalicious()
+		initiallyMalfeasant = true
 	}
```

Comment:
NIT: `initiallyMalfeasant` isn't needed anymore:
```suggestion
	malicious := msh.atxsdata.IsMalicious(ballot.SmesherID)
	if malicious {
		ballot.SetMalicious()
	}

	start := time.Now()
	// use `malicious` in place of `initiallyMalfeasant`
```

Reply:
applied via extra commit. thanks for the thorough review @poszu